### PR TITLE
Custom line ending support

### PR DESCRIPTION
**Pull request by martonpe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,11 +49,12 @@ This will uncompress the file and converts it to avro, leaving the original file
 For a full list of available options, run `csv2avro --help`
 ```
 $ csv2avro --help
-Version 1.0.1 of CSV2Avro
+Version 1.3.0 of CSV2Avro
```
Comment (gszjulcsi) [inline README.md]: How come? :)

Comment (martonpe) [inline README.md]: Well, that's the next upcoming version, we're at `1.2.0` now. I just thought I'd bump, it doesn't really matter.

File: spec/support/data_crlf.csv [hunk 2]
```diff
@@ -0,0 +1,4 @@
+id,name,description
+1,dresses,Dresses
+"2","female-tops",
+"3",,"Bras"
```
Comment (gszjulcsi) [inline spec/support/data_crlf.csv]: Please add a new line at the end of the file, unless it breaks the test

**Comment by gszjulcsi** [comment]
LGTM

File: spec/support/data_crlf.csv (hunk 2 above)
Comment (martonpe) [inline spec/support/data_crlf.csv]: If it would, that would be a problem. :smile: Added.

**Pull request merged**